src/RageSoundPosMap.cpp

```diff
@@ -181,12 +162,7 @@ std::int64_t pos_map_queue::Search( std::int64_t iSourceFrame, bool *bApproximat
 	static RageTimer last;
```

Comment:
We can delete this definition too since we don't use it.

Reply:
We need this `RageTimer` definition to use `PeekDeltaTime()`.

If we get rid of it, we could log that the closest position is being returned and not do the `PeekDeltaTime()` check.

Reply:
OH I totally missed that condition. Wait in that case we *do* need the Touch() just to prevent it from logging really fast.

Reply:
😅  Putting `last.Touch()` back.